pandas/io/orc.py

```diff
@@ -47,6 +47,8 @@ def read_orc(
         ``file://localhost/path/to/table.orc``.
     columns : list, default None
         If not None, only these columns will be read from the file.
+        Output always follows the ordering of the file and not the columns list.
+        This mirrors the original behaviour of :func:`pyarrow.orc.OrcFile.read`.
```

Comment:
```suggestion
        This mirrors the original behaviour of :external+pyarrow:py:func:`pyarrow.orc.OrcFile.read`.
```

Reply:
@mroeschke @phofl fixed. I had to change `py:func:` to `py:meth:`. Now the link opens correctly